controllers/articles-controller.js

```diff
@@ -0,0 +1,14 @@
+const { selectArticleById } = require("../models/articles-model");
+
+exports.getArticleById = (request, response, next) => {
+    const { article_id } = request.params;
+    selectArticleById(article_id)
+        .then((article) => {
+            response.status(200).send({ article })
+        })
+        .catch((err) => {
+            if (err) {
```

Comment:
you don't have to use an if statement here, you can just pass the error to your next function